packages/gatsby/src/internal-plugins/query-runner/query-compiler.js

```diff
@@ -217,16 +233,13 @@ class Runner {
     return compiledNodes
   }
 }
-export { Runner }
+export { Runner, resolveThemes }
 
 export default async function compile(): Promise<Map<string, RootQuery>> {
-  const { program, schema } = store.getState()
+  // TODO: swap plugins to themes
```

Comment:
cc @ChristopherBiscardi once #10787 lands I can swap to `themes` from the redux state.

I don't really consider it a blocker for this PR though. We can get this one merged and then tweak later, removing the `gatsby-theme` test which will be 👌

Reply:
sgtm